### PR TITLE
Create Problem_Statement_57_Solution.java

### PR DESCRIPTION
**Pull request by yashasvi17rawat**

### CODE REVIEW

File: Solutions/Problem_Statement_57_Solution.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Write a program to print the names of students by creating a Student class. If no name is passed
+while creating an object of Student class, then the name should be "Unknown", otherwise the name
+should be equal to the String value passed while creating object of Student class.
+....
+*/
+class Hack//creating a class hack
+{
+	public static void main(String[] args)
+	{
+		Student obj1=new Student("Riya");//created an object of Student class while passing parameter as name
```
Comment (Krush2311) [inline Solutions/Problem_Statement_57_Solution.java:12]: @yashasvi17rawat We get an output: 
Riya
Unknown
which is correct!
But I would like to have an user defined program. Could you please do the needful changes?

**Review by Krush2311 — CHANGES_REQUESTED** [review]

**Comment by yashasvi17rawat** [comment]
For user defined can I use switch case to give a choice to user if it wants
to enter a name or not and then respectively print the name or unknown as
an output..?

After the changes Ill again have to give pull request?

Thanking you.

**Comment by Krush2311** [comment]
No! What I expect is:,
Output:
Enter name:
(User enters name and presses enter .After this the output is the input given by the user. But if nothing is provided and only enter is pressed then output is "unknown ")

**Comment by yashasvi17rawat** [comment]
i have made the changes and created another pull request yesterday.

**Comment by Krush2311** [comment]
> i have made the changes and created another pull request yesterday.
Okay

**Comment by Shr03mink** [comment]
@yashasvi17rawat you don't have to create a new PR every time you make a change in code. simple push new changes and they will be rebased with the earlier commit. For a detailed info, go through our Revision Video that we shared the day before yesterday

**Pull request closed without merging**